notebooks/22.Unsupervised_learning-anomaly_detection.ipynb

```diff
@@ -733,7 +733,7 @@
     "\n",
     "fig, axes = plt.subplots(2, 5, figsize=(10, 5))\n",
     "\n",
-    "for i, ax in zip(range(len(X_outliers)), axes.ravel()):\n",
```

Comment:
maybe @agramfort wants to check this out?

Reply:
bug fix ok correct. good catch @mppaskov